funnel/views/commentvote.py

```diff
@@ -187,15 +187,13 @@ def subscribe(self):
             db.session.commit()
             return {
                 'status': 'ok',
-                'message': _("You have successfully subscribed to this comment thread"),
+                'message': _("Successfully subscribed to this comment thread"),
             }
         else:
             return {
                 'status': 'error',
                 'error_code': 'subscribe_error',
-                'error_description': _(
-                    "Invalid CSRF token. Reload the page and try again"
-                ),
+                'error_description': _("This page timed out. Reload and try again"),
```

Comment:
We're using "refresh" elsewhere. Need to use one term. Chrome and Firefox both use "reload" so maybe that should too. Can you fix the places it says "refresh"? (Not necessarily in this PR.)

Reply:
Sure.